CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 - Fix Guillotina tests @bloodbare
 - Fix problem with not wrapped element in `Provider` store in `WysiwygWidget` component
   due that now, the links are wrapped with a connected component @sneridagh
+- Responsive header @ksuess
```

Comment:
While doing the other changes, can you update it and put it into the right position?